packages/babel-plugin-transform-computed-properties/test/fixtures/regression/15140/exec.js

```diff
@@ -0,0 +1,3 @@
+var x = { x, get x() { return 0; }, x };
```

Comment:
> packages/babel-plugin-transform-computed-properties/test/fixtures/regression/15140/exec.js

This doesn't appear to be a regression.

Reply:
`spec/definition-ordering` looks good to me, also `loose/definition-ordering` if it works.

Reply:
Oops, seems that I misunderstood the meaning of "regression". Then the name of the tests in my previous PR #15183 and #15194 may also be wrong...

Reply:
Now the test is renamed to `spec/definition-order`. It doesn't apply to `loose` or `assumption-setComputedProperties`.